Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -136,8 +136,14 @@ def set_data(self, data):
         self.set_input_summary()
 
         if data is not None and data.domain.class_var is None:
-            self.Error.data_error("Data has no target variable.\n"
-                                  "You can set a target variable with the Select Columns widget.")
+            if data.domain.class_vars:
+                self.Error.data_error(
+                    "Data contains multiple target variables.\n"
+                    "You can select a single one with Select Columns widget.")
+            else:
+                self.Error.data_error(
+                    "Data has no target variable.\n"
+                    "You can select one with Select Columns widget.")
```

Comment:
the Select Columns widget

Reply:
I intentionally removed `the` to make it shorter. But OK, I added it back.

Reply:
Honestly, I didn't open the widget and check. Generally, I'm not too worried about the message length, they're visible on hover/click.